src/DotVVM.Framework/Resources/Scripts/postback/postback.ts

```diff
@@ -68,7 +68,7 @@ export async function postBack(
                 serverResponseObject,
                 wasInterrupted,
                 commandResult: null,
-                response: (err.reason as any).response,
+                response: (err.reason as any)?.response,
```

Comment:
When an exception is thrown the reason si null. That's why null propagation is needed.

Reply:
Exception on the server? That would be `type: "serverError"`. Exception on the client should also contain the reason when it's of type `DotvvmPostbackError`. `DotvvmPostbackError`'s type definition does not allow null, so could either update it or (preferably IMHO) fix the bug that's causing null in there?

Reply:
The null in `reason` was caused by an exception in `dotvvm.events.<>.subscirbe((e)=>{ "some error here" })`. In this case, you need to expect that it can be null. It is non-dotvvm code that will result in this exception and dotvvm has to work with it. 
The type definition works only for typescript. But not everyone has to be using it.

Reply:
In that case, I'd better add a null check to the constructor. If you terribly need to be null-tollerant, I'd suggest changing the type signature so we can at least be consistent. Even with your additional null checks, the handler is going to crash in `isInterruptingErrorReason`

Reply:
@quigamdev I think we should add try/catch blocks around errors in event handlers and wrap the error with `reason: event`. But I think we can do it in a separate PR.